src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -660,6 +660,12 @@ bool QgsPostgresConn::getTableInfo( bool searchGeometryColumnsOnly, bool searchP
   result = LoggedPQexec( "QgsPostgresConn", query );
   // NOTE: we intentionally continue if the query fails
   //       (for example because PostGIS is not installed)
+
+  if ( ! result.result() )
+  {
+    return true;
```

Comment:
Is true the correct thing to return here?

Reply:
Probably not.